### ISSUE TITLE
Feature request: Add agile accessibility reference

### ISSUE THREAD

**Issue by lukefretwell** [CivicActions/accessibility#338]
**Is your feature request related to a problem? Please describe.**

@mgifford used the term 'agile accessibility' as the approach we should be using, so it would be great to document this so we have a shared understanding of what it is, but also a single source of truth when socializing this internally/externally.

**For whom is this feature?**
A clear and concise user story.
As a CivicActioner, I want to have a single source of truth on 'agile accessibility', so that I can better understand what it is, but also share that with others.
Personas: Accessibility Avery, Designer Darcy, Engineer Emery, Procurement Parker, QA Quincy, Remote Raz, and CivicActioner

**Describe the solution you'd like**

* Page in guide
* Blog post

**Additional context**

* https://accessibility.deque.com/agile-accessibility-handbook
* https://www.amazon.com/Agile-Accessibility-Explained-sustainable-development-ebook/dp/B07Z38ZCJ3
* https://karlgroves.com/2011/09/28/agile-accessibility
* https://www.youtube.com/watch?v=cUovTdJOWu8

**Comment by mgifford** [comment]
- Adoption of agile requires a change in processes for the whole team.
- The process has a clear understanding of [roles & responsibilities](https://accessibility.civicactions.com/playbook/roles)
- Automated testing is an important part of ensuring quality & making sure that bugs are caught as early as possible there is a layered approach that addresses [page level, pattern level & site-wide testing](https://accessibility.civicactions.com/playbook/automated-testing)
- The definition of done needs to include [manual testing](https://accessibility.civicactions.com/playbook/manual-testing) including testing with [assistive technology](https://accessibility.civicactions.com/playbook/AT)
- [Training](https://accessibility.civicactions.com/playbook/training) is something that will need to be an ongoing part of the team, and ultimately [people with disabilities](https://accessibility.civicactions.com/playbook/pwd) need to be involved in the process at critical phases to see that the goals & deliverables do not exclude PwD.
